flang/include/flang/Frontend/CompilerInstance.h

```diff
@@ -133,6 +136,16 @@ class CompilerInstance {
     semantics_ = std::move(semantics);
   }
 
+  void setRtTyTables(
```

Comment:
Some of these member functions above have doxygen comments. These do not.

Reply:
Added